examples/deployment_ec2_instance/aws_ec2.tf

```
@@ -0,0 +1,37 @@
+resource "aws_instance" "inst" {
```

I feel like `inst` doesn't say much. How about changing it to `"web"` or something like that? 
```suggestion
resource "aws_instance" "web" {
```